### ISSUE TITLE
Plot starts at zero on left side

### ISSUE THREAD

**Issue by Jornan** [philackm/ScrollableGraphView#113]
When I create a graph there is a straight line drawn from the left most visible data point to a point at zero right outside the bounds of the view. The right side is fine with no issues. Anyone else seing this issue too? See screenshots for three examples.
![grapherror](https://user-images.githubusercontent.com/3278797/31584551-be84df5a-b1b0-11e7-8520-c90429c4d3c5.jpg)

**Comment by ohserezha** [comment]
yes. it actually starts with zero for any sequence of values. sorry, i don't actually remember how i resolved it, but u should look at leftMostPointPadding and other offsets

**Comment by wanglouin1990925** [comment]
same issue for me. is there any solution for this issue?

**Comment by jeffreality** [comment]
It's because of the fill. Try it with `plot.shouldFill = false` - you will lose the color under your graph, but it will fix that particular issue. I just figured this out myself a few minutes ago.

**Comment by philackm** [comment]
As @jeffreality has stated, this is occurring because of the fill underneath the line. This was made as a design decision as for what I needed it looked slightly better.

We would need to change the way the path is generated depending on some property. For example, instead of extending the path down to "zero" at the start, we would want to draw the path straight down. See the image as an example:

<img width="851" alt="screen shot 2018-05-15 at 9 23 37 pm" src="https://user-images.githubusercontent.com/2552214/40054021-b8d69daa-5886-11e8-8b0b-9300d94486b9.png">

The current workaround is to not fill the graph.
